mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/bridge/runtime/impl/ProvidedStringIdentifierMapping.java

```diff
@@ -9,21 +9,36 @@
 import java.lang.invoke.MethodHandles;
 import java.util.function.Supplier;
 
+import org.hibernate.search.engine.environment.bean.BeanHolder;
+import org.hibernate.search.mapper.pojo.bridge.IdentifierBridge;
 import org.hibernate.search.mapper.pojo.bridge.runtime.spi.BridgeMappingContext;
 import org.hibernate.search.mapper.pojo.bridge.runtime.spi.BridgeSessionContext;
 import org.hibernate.search.mapper.pojo.logging.impl.Log;
+import org.hibernate.search.util.common.impl.Closer;
 import org.hibernate.search.util.common.logging.impl.LoggerFactory;
 
 
 public class ProvidedStringIdentifierMapping implements IdentifierMappingImplementor<String, Object> {
```

Comment:
The type arguments are wrong here. It should be `<Object, Object>`. See the javadoc of the interface.

Also, consequently, you should probably remove the class to `ProvidedIdentifierMapping`: it's no longer exclusively about string identifiers.

Reply:
You're definitely right. Do you mean `you should probably` **rename** `the class`?

Reply:
Yes, that was a typo.